b2/account_info.py

```diff
@@ -0,0 +1,318 @@
+######################################################################
+#
+# File: b2/account_info.py
+#
+# Copyright 2016 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+import json
+import os
+import stat
+from abc import (ABCMeta, abstractmethod)
+
+import six
+
+from .exception import (MissingAccountData)
```

Comment:
I think yapf adds parens when it goes multi-line, and doesn't remove them when i shrinks.